DQ - Surviving Record/README.md

```diff
@@ -2,36 +2,46 @@
  
 ## Description
 
-The **DQ - Surviving Record** step allows you to extract the best record (Golden Record) from the clustered data by selecting the rules of interest. You can use standard deduplication techniques or create a rule to choose which data should survive.
-
+The **DQ - Surviving Record** step allows you to extract the best record (Golden Record) from the clustered data by selecting the rules of interest. You can use standard deduplication techniques or create a rule to choose which data should survive.  
+This new version uses the standard macro [%DQSURVR](https://go.documentation.sas.com/doc/en/dqcdc/v_025/dqclref/p1i32jjeybfcb7n19tzfasbzdcb9.htm) instead of the custom code.  
```

Comment:
Always uses doc URLs that point to the latest doc, using .../default/... instead of .../v_xxx/...
```suggestion
This new version uses the standard macro [%DQSURVR](https://go.documentation.sas.com/doc/en/dqcdc/default/dqclref/p1i32jjeybfcb7n19tzfasbzdcb9.htm) instead of the custom code.  
```